.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+---
+
+**Problem Description**
+<!-- Please add a clear and concise description of what the problem is. -->
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
I think the comment and this statement is repetitive. What do you think? We should probably remove it.

Reply:
yes, thats what i was thinking. the heading  and comment is self explanatory. Ill remove the statement then.